osu.Game/Graphics/OsuColour.cs

```diff
@@ -20,26 +22,41 @@ public Color4 ForDifficultyRating(DifficultyRating difficulty, bool useLighterCo
             switch (difficulty)
             {
                 case DifficultyRating.Easy:
-                    return Green;
+                    return Color4Extensions.FromHex("4ebfff");
 
-                default:
                 case DifficultyRating.Normal:
-                    return Blue;
+                    return Color4Extensions.FromHex("66ff91");
 
                 case DifficultyRating.Hard:
-                    return Yellow;
+                    return Color4Extensions.FromHex("f7e75d");
 
                 case DifficultyRating.Insane:
-                    return Pink;
+                    return Color4Extensions.FromHex("ff7e68");
 
                 case DifficultyRating.Expert:
-                    return PurpleLight;
+                    return Color4Extensions.FromHex("fe3c71");
 
                 case DifficultyRating.ExpertPlus:
-                    return useLighterColour ? Gray9 : Color4Extensions.FromHex("#121415");
+                    return Color4Extensions.FromHex("6662dd");
```

Comment:
How were these derived, exactly?

Reply:
It seems there's no direct references to this, but I took it directly from the strips on the beatmap picker in the redesigned beatmap info page.

osu-web also uses the same colours, minus one inaccuracy in `DifficultyRating.Hard` that I need to check: https://github.com/ppy/osu-web/blob/71fbab8936d79a7929d13854f5e854b4f383b236/resources/assets/less/variables.less#L308-L316

Reply:
That link to web needs to be in source, then, for posterity.

Reply:
It seems the designs themselves are being inconsistenton the colour for `DifficultyRating.Hard`, I'm just gonna match with osu-web for now.